packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue

```diff
@@ -1,27 +1,46 @@
 <template>
 	<aside class="overlay-container">
 		<section>
-			<header ref="header">
-				<div>
-					<slot name="header" />
-				</div>
-				<Button
-					icon="pi pi-times"
-					text
-					rounded
-					aria-label="Close"
-					@click="emit('on-close-clicked')"
-				/>
-			</header>
-			<main><slot /></main>
+			<tera-drilldown-header
+				:active-index="selectedViewIndex"
+				:views="views"
+				:tooltip="tooltip"
+				@tab-change="handleTabChange"
+				@close="emit('on-close-clicked')"
+				>{{ props.title }}</tera-drilldown-header
+			>
+			<main>
+				<template v-for="(view, index) in views">
+					<slot v-if="selectedViewIndex === index" :name="view" />
+				</template>
+			</main>
 		</section>
 	</aside>
 </template>
 
 <script setup lang="ts">
-import Button from 'primevue/button';
+import TeraDrilldownHeader from '@/components/drilldown/tera-drilldown-header.vue';
+import { TabViewChangeEvent } from 'primevue/tabview';
+import { computed, onMounted, ref, useSlots } from 'vue';
+
+const props = defineProps<{
+	title: string;
+	tooltip?: string;
+}>();
 
 const emit = defineEmits(['on-close-clicked']);
+const slots = useSlots();
+
+const views = computed(() => Object.keys(slots));
```

Comment:
I'm going to give this one a think over when I start with the main container component.  The jist of this is you can pass in as many slots as you want into the drilldown and each will get their own tab view.

```
<tera-drilldown>
    <template #Wizard>
        <!--- dev adds wizard components here -->
    </template>
    <template #Notebook>
        <!--- dev adds notebook components here -->
    </template>
    <!-- ...dev can add as many templates as they want for each tab -->
</tera-drilldown>
```